code/game/objects/items/devices/sound_synth.dm

```diff
@@ -65,6 +65,10 @@
         playsound(src, selected_sound, volume, shiftpitch)
         spam_flag = world.timeofday
 
+/obj/item/soundsynth/AltClick(mob/living/carbon/user, obj/item/I)
```

Comment:
The AltClick proc has no item parameter, only a user.

Reply:
Huh, I was basing this off the paperplane code. Seems there I is overridden. Who writes this stuff?